### PR TITLE
PT-5940 Expose version item

### PR DESCRIPTION
**Pull request by sanufriev**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -3216,6 +3218,64 @@ paths:
...
 
+
+  ######################### Reconciliation API ####################
+
+  /v0.1/reconciliation/items:
+    get:
+      tags:
+        - "Reconciliation"
+      summary: "Get IDs of items updated after a specific datetime."
+      description: "Return user's balance of specified currency"
```
Comment (EduardovEduard) [inline openapi.yaml:3229]: The description seems wrong

File: openapi.yaml [hunk 2]
```diff
@@ -3216,6 +3218,64 @@ paths:
...
+      operationId: getReconciliationItems
+      parameters:
+        - name: "blockchain"
```
Comment (igorbaltiyskiy-rari) [inline openapi.yaml:3232]: Why do we need the blockchain parameter? Isn't it easier to reconciliate for all blockchains simultaneously?

File: pom.xml [hunk 3]
```diff
@@ -34,7 +34,7 @@
     <version>${revision}</version>
 
     <properties>
-        <revision>1.105.100</revision>
+        <revision>1.105.101</revision>
```
Comment (igorbaltiyskiy-rari) [inline pom.xml:37]: This version is already used in the release branch

https://github.com/rarible/union-openapi/blob/release/1.105/pom.xml#L37

Seems like someone messed up when setting version in main

Comment (igorbaltiyskiy-rari) [reply pom.xml:37]: Yes, it was me 🤦

Comment (igorbaltiyskiy-rari) [reply pom.xml:37]: I suggest we fix it and set 1.105.2

Comment (igorbaltiyskiy-rari) [reply pom.xml:37]: Ah, OK, so I was right after all

**Pull request merged**